### PR TITLE
Added the Git exporter

### PR DESCRIPTION
**Pull request by rievo**

### CODE REVIEW

File: app/exporters/git.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+                    "remote",
+                    "add",
+                    "origin",
+                    self.get_auth_export_url_from_repo(
+                        job.export_url, job.export_token
+                    ),
+                ],
+                cwd=path_to_local_files,
+            )
+        except subprocess.CalledProcessError:
```
Comment (squeaky-pl) [inline app/exporters/git.py:154]:
```suggestion
        except subprocess.CalledProcessError as e:
```

File: app/exporters/git.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+            raise GitRemoteOriginError("Found an error while setting the remote origin")
```
Comment (squeaky-pl) [inline app/exporters/git.py:155]:
```suggestion
            raise GitRemoteOriginError("Found an error while setting the remote origin") from e
```

File: app/exporters/git.py [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+            cwd=path_to_local_files,
+        )
+
+    def git_command_push_to_remote(self, path_to_local_files: str) -> None:
+        try:
+            subprocess.check_output(
+                ["git", "push", "-u", "origin", "main"],
+                cwd=path_to_local_files,
+            )
+        except subprocess.CalledProcessError:
```
Comment (squeaky-pl) [inline app/exporters/git.py:169]:
```suggestion
        except subprocess.CalledProcessError as e:
```

File: app/exporters/git.py [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+            raise GitPushError("Error pushing to origin/main")
```
Comment (squeaky-pl) [inline app/exporters/git.py:170]:
```suggestion
            raise GitPushError("Error pushing to origin/main") from e
```

**Review by squeaky-pl — CHANGES_REQUESTED** [review]

File: app/service_validators/services.py [hunk 5]
```diff
@@ -7,7 +7,7 @@
     regex_validator,
     service_id="git",
     regexes=[
-        r"^https?:\/\/(www\.)?git(hub|lab)\.com\/(?P<organization>[\w-]+)/(?P<project>[\w-]+)"
+        r"^https?:\/\/(www\.)?git(?P<service>hub|lab)\.com\/(?P<user>[\w-]+)/(?P<project>[\w-]+)"
```
Comment (drewler) [inline app/service_validators/services.py:10]: Guess even if not 100% accurate, `user` is better than `organization`

**Review by squeaky-pl — APPROVED** [review]

**Pull request merged**